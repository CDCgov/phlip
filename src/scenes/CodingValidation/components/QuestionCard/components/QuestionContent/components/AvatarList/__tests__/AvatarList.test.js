import React from 'react'
import { shallow } from 'enzyme'
import { AvatarList } from '../index'

const props = {
  answerList: [],
  userImages: {},
  selectedIndex: 0,
  handleClickAvatar: jest.fn()
}

describe('QuestionCard - QuestionContent - AvatarList', () => {
  test('should render correctly', () => {
    expect(shallow(<AvatarList {...props} />)).toMatchSnapshot()
  })
})
