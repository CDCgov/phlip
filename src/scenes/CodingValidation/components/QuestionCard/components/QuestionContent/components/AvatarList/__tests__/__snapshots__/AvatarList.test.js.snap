// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QuestionCard - QuestionContent - AvatarList should render correctly 1`] = `
<FlexGrid
  align="center"
  circular={false}
  container={true}
  flex={false}
  justify="stretch"
  padding={0}
  raised={false}
  style={Object {}}
  type="row"
>
  <WithStyles(Tooltip)
    text="Show all annotations"
  >
    <Avatar
      alt=""
      avatar=""
      big={false}
      cardAvatar={false}
      initials="ALL"
      key="user-avatar-all-selected"
      style={
        Object {
          "backgroundColor": "#3d316a",
          "color": "white",
        }
      }
      theme={Object {}}
      userName="All"
    />
  </WithStyles(Tooltip)>
</FlexGrid>
`;
